### PR TITLE
Fix return type in supports fullscreen

### PR DESCRIPTION
**Pull request by Davoud-Ghargozlou**
Hi  
I refactored [this issue](https://sonarcloud.io/project/issues?open=AYjJwpl65aesmyw3FN6K&id=jellyfin_jellyfin-web) on SonarCloud

### CODE REVIEW

**Comment by silentTeee** [comment]
I think your commit from #4902 is part of this PR as well. Was that intentional?

**Comment by thornbill** [comment]
Tagged as "blocked" since this includes unrelated changes.

File: src/components/apphost.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ function supportsFullscreen() {
     }
 
     const element = document.documentElement;
-    return (element.requestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen || element.msRequestFullscreen) || document.createElement('video').webkitEnterFullscreen;
+    return !!(element.requestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen || element.msRequestFullscreen) || document.createElement('video').webkitEnterFullscreen;
```
Comment (seeplusplus) [inline src/components/apphost.js:150]:
```suggestion
    return !!((element.requestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen || element.msRequestFullscreen) || document.createElement('video').webkitEnterFullscreen);
```

Comment (dmitrylyzo) [reply src/components/apphost.js:150]: There is no need for additional parentheses.
```suggestion
    return !!(element.requestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen || element.msRequestFullscreen || document.createElement('video').webkitEnterFullscreen);
```

**Review by thornbill — APPROVED** [review]

**Pull request merged**